pkg/cmdimpl/add.go

```diff
@@ -88,6 +88,11 @@ func updateParametersIfNecessary(gitClient git.Git) error {
 		params.DeploymentType = string(DeployTypeHelm)
 	}
 
+	if params.AppConfigUrl == "" && params.SourceType != string(SourceTypeGit) {
+		return fmt.Errorf("Cannot create .wego directory in helm repository:\n" +
```

Comment:
Nit: error message should start with a lower case letter (go style guide)

Reply:
Good catch